.github/workflows/integrate.yaml

```
@@ -31,6 +31,10 @@ jobs:
       - name: Add Firebase Messaging
         run: ./scripts/add-firebase-messaging.sh
       - run: flutter analyze
+      - name: Make .env file
++       run: |
++         echo "TEST_MATRIX_USERNAME=${{ secrets.TEST_MATRIX_USERNAME }}" >> .env
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)